### PR TITLE
Add support for multi-srcpath

### PR DESCRIPTION
**Pull request by daltonmatos**
Now Vebugger can search for a sourcecode in multiple srcpath locations.
The first match is returned and it's the file Vebugger will show.

### CODE REVIEW

**Comment by idanarye** [comment]
Any reason not to simple let `srcpath` support lists?

**Comment by daltonmatos** [comment]
It's a possibility. I kept it as a string for simplicity, also the split just works even if the `srcpath` does not contain `:`.

Do you think it's better to support both string (for single path) and list (for multi path)? I could adjust the code, no problem.

**Comment by idanarye** [comment]
Yes - support both strings and lists. You can add something like this to `autoload/vebugger/util.vim`:

```vim
function! vebugger#util#listify(stringOrList)
    if type(a:stringOrList) == type([])
        return copy(a:stringOrList) " so it could safely be modified by map&filter
    else
        return [a:stringOrList]
    endif
endfunction
```

**Comment by daltonmatos** [comment]
Done. Waiting for review.

Thanks,

File: autoload/vebugger/jdb.vim [hunk 1]
```diff
@@ -87,14 +87,16 @@ function! s:findFolderFromStackTrace(src,nameFromStackTrace,frameNumber)
...
-	let l:path=a:src
-	for l:dirname in split(a:nameFromStackTrace,'\.')
-		let l:nextPath=l:path.'/'.fnameescape(l:dirname)
-		if empty(glob(l:nextPath))
-			return l:path
-		endif
-		let l:path=l:nextPath
-	endfor
+  for l:one_path in vebugger#util#listify(a:src)
+    let l:path=l:one_path
```
Comment (idanarye) [inline autoload/vebugger/jdb.vim]: You could probably just use `l:path` as the iteration variable, but OK.

File: doc/vebugger.txt [hunk 2]
```diff
@@ -202,14 +202,18 @@ Unlike in the other debuggers, the first argument here is not the name of a
 file - it's the name of the class to run. The supported extra arguments are:
 * "args": Command line arguments for the debugged program
 * "classpath": Where to look for class files
-* "srcpath": Where to look for source files
+* "srcpath": Where to look for source files. You can have multiple srcpaths, separated by ':'
```
Comment (idanarye) [inline doc/vebugger.txt]: The docs need updating, too

File: autoload/vebugger/jdb.vim (hunk 1 above)
Comment (idanarye) [inline autoload/vebugger/jdb.vim]: Also... please use 4 spaces for indentation.

File: doc/vebugger.txt (hunk 2 above)
Comment (daltonmatos) [reply doc/vebugger.txt]: Done. Will push soon.

File: autoload/vebugger/jdb.vim (hunk 1 above)
Comment (daltonmatos) [reply autoload/vebugger/jdb.vim]: Done.

Comment (daltonmatos) [reply autoload/vebugger/jdb.vim]: The file seems to be using tab-indentation. Do you want to expand these tabs to 4-spaces, or should I just change the spaces I created with tabs?

Comment (idanarye) [reply autoload/vebugger/jdb.vim]: Never mind then - just fix the docs and I'll `:retab` it myself.

**Comment by daltonmatos** [comment]
Docs updated.

**Pull request merged**